src/core/viz/expressions/top.js

```diff
@@ -1,5 +1,23 @@
 import BaseExpression from './base';
 
+/**
+ * Get the top `n` properties
+ *
+ * @param {carto.expressions.property} property - Column of the table
+ * @param {carto.expressions.number} n - Number of top properties to be returned
```

Comment:
I think this must be a JS Number, not a number expr.

Reply:
Yes